tools/metadata_tools/process_metadata.py

```diff
@@ -235,8 +181,6 @@ def main():
                     print(f"skipping path; does not exist: {path_to_readme}")
                     continue
                 sample.populate_from_readme(platform, path_to_readme)
-                if platform == "FormsAR":
```

Comment:
We shouldn't do this same thing with the maui AR samples. They should be put in normal category and we can just have platform specific exclusions for MacCatalyst and WinUI.

Reply:
I need some clarification here.